### DISCUSSION TITLE
Existing MAUI project unable to run on MacOS

### DISCUSSION THREAD

**Discussion by achobanov** [OmniSharp/omnisharp-vscode#7400] [Q&A] [answered]
I have an existing project that I'm able to run from VS Code running on Windows, but not on MacOS. After a whole day of wrestling with MAUI stuff on dotnet 7, I've updated all projects to target net8 and now the project builds successfully, however it still cannot start it up, because VS Code attempts to start it:

>launch: program '<project-path>/bin/Debug/net8.0-maccatalyst/App' does not exist. 

I'm pretty sure VS Code is looking at the wrong path for some reason, because there are multiple types of `maccatalyst` targets. In this case the actual path where you can find the dlls is:
```
<project-path>/bin/Debug/net8.0-maccatalyst/maccatalyst-x64/App
```

I'm able to startup a fresh template maui solution no problem.

**Comment by gregg-miskelly** [comment, marked as answer]
Since you have a MAUI app, I am assuming you are attempting to launch using the .NET MAUI extension? If so, you want to ask at https://github.com/microsoft/vscode-dotnettools/issues

**Reply by achobanov** [reply]
That makes sense, I had a mushy head at the end of that day.. Here's a link to the issue here, in case someone stumbles upon this topic: https://github.com/microsoft/vscode-dotnettools/issues/1324
